### PR TITLE
Create ui-c8y-1019-24-5-user-message-in-device-list-page-is-incorrect-when-no-devices.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by bode

Ticket: [DM-1214](https://cumulocity.atlassian.net/browse/DM-1214)
Original PR: [6389](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6389)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1019-24-5-user-message-in-device-list-page-is-incorrect-when-no-devices.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: User message in device list page is incorrect when no devices are available in the tenant [GRAFT][release/cd]
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-24-5-user-message-in-device-list-page-is-incorrect-when-no-devices.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Improved user message on device list page when no devices are available
```

File: content/change-logs/device-management/ui-c8y-1019-24-5-user-message-in-device-list-page-is-incorrect-when-no-devices.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-1214
+version: 1019.24.5
+---
+User message in device list page is incorrect when no devices are available in the tenant [GRAFT][release/cd]
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1019-24-5-user-message-in-device-list-page-is-incorrect-when-no-devices.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, when no devices were available in a tenant, the device list page displayed an incorrect user message. With this change, the user message on the device list page has been updated to provide a clear and accurate message when no devices are available in the tenant. This improvement enhances the user experience by providing precise feedback on the device availability status.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@bozhidardedov This seems to be a duplicate of #2075?

**Comment by bozhidardedov** [comment]
> @bozhidardedov This seems to be a duplicate of #2075?

It is a duplicate. We reverted the first PR and merged the changes later. Does it matter which of the two CL PRs gets merged? If we can use the first one, I'll close this one.

**Comment by BeateRixen** [comment]
> > @bozhidardedov This seems to be a duplicate of #2075?
> 
> It is a duplicate. We reverted the first PR and merged the changes later. Does it matter which of the two CL PRs gets merged? If we can use the first one, I'll close this one.

No, it doesn´t  matter.

**Comment by bozhidardedov** [comment]
Duplicate of https://github.com/SoftwareAG/c8y-docs/pull/2075

**Pull request closed without merging**